modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -1116,49 +1110,58 @@ module.exports = (self) => {
       }
       if (!clauses.length) {
         destination[field.name] = [];
-        return;
```

Comment:
Just nesting this in an if/else so I can look at the end result in one place, all this logic hasn't really changed.